### PR TITLE
Revert "Adds a fumble to baseline pill bottles."

### PR DESCRIPTION
**Pull request by Lumipharon**
Reverts tgstation/TerraGov-Marine-Corps#14296

image

### CODE REVIEW

**Comment by lKiseki** [comment]
spooky i know you're here you should instantly speedmerge this pr

**Review by spookydonut — APPROVED** [review]

File: code/game/objects/items/storage/pill_packets.dm [hunk 1]
```diff
@@ -66,7 +65,7 @@
 	pip_color = COLOR_PACKET_DYLOVENE
 
 /obj/item/storage/pill_bottle/packet/paracetamol
-	name = "paracetamol pill packet"
+	name = "paracematol pill packet"
```
Comment (Xander3359) [inline code/game/objects/items/storage/pill_packets.dm:68]: Probably not a good idea to re-introduce typos

**Review by Xander3359 — CHANGES_REQUESTED** [review]

**Comment by lKiseki** [comment]
HOLY BASED

File: code/game/objects/items/storage/pill_packets.dm (hunk 1 above)
Comment (Lumipharon) [reply code/game/objects/items/storage/pill_packets.dm:68]: yous a tipo

Comment (spookydonut) [reply code/game/objects/items/storage/pill_packets.dm:68]: a result of non-atomic changes

**Pull request merged**